core/src/main/java/com/oath/oak/ValueUtils.java

```diff
@@ -137,15 +110,15 @@
      * Used to try and read a value off-heap
      *
      * @param result      The result object
-     * @param s           the value's off-heap object
+     * @param value       the value's off-heap object
```

Comment:
off-heap object --> buffer
everywhere

Reply:
I changed it as per your request. But please note that it technically not a buffer. It is a `Slice`.
If we want to align ourselves with referring the off-heap handlers as buffers, we might want to rename `Slice` to `OffheapBuffer`.

Reply:
Actually you are right it needs to be Slice and everywhere...

Reply:
Eventually, we will have to find a more suitable name for `Slice`.
I don't really understand what was wrong with the name you suggested in the paper: `Handler`, or `OffheapHandler`.
But for now, I'll change the comment to `the value's off-heap Slice object`.

Reply:
OK. Slice is used in the Nova paper. I like this name and want to keep it. But this is not related here.